packages/lab/src/stepped-tracker/StepLabel/StepLabel.tsx

```diff
@@ -1,51 +1,26 @@
-import {
-  forwardRef,
-  ReactNode,
-  useMemo,
-  ComponentPropsWithoutRef,
-} from "react";
+import { forwardRef, ComponentPropsWithoutRef } from "react";
 import { clsx } from "clsx";
-import { useForkRef, makePrefixer, useIdMemo, Label } from "@salt-ds/core";
+import { makePrefixer, Label } from "@salt-ds/core";
 import { useComponentCssInjection } from "@salt-ds/styles";
 import { useWindow } from "@salt-ds/window";
 
 import stepLabelCss from "./StepLabel.css";
 
-import { useSteppedTrackerContext } from "../SteppedTrackerContext";
-import { useTrackerStepTooltipContext } from "../TrackerStep";
-
 const withBaseName = makePrefixer("saltStepLabel");
 
-export interface StepLabelProps extends ComponentPropsWithoutRef<"label"> {
-  children: ReactNode;
-}
+export type StepLabelProps = ComponentPropsWithoutRef<"label">;
 
 export const StepLabel = forwardRef<HTMLLabelElement, StepLabelProps>(
   function StepLabel({ children, className, ...rest }, ref?) {
```

Comment:
Just a question, what is the question mark for?

Reply:
I don't think it does anything here as ForwardedRef can already be null. Not sure why I added it, might have cut and paste the forwardRef from another component